### DISCUSSION TITLE
utm for linux (debian)

### DISCUSSION THREAD

**Discussion by mjonsson1986** [utmapp/UTM#3678] [UTM Help (macOS)] [answered]
not an issue but a question
are there a utm for debian linux?
because under the assets there are a deb file. deb files are for debian linux..
**Describe the issue**
A clear and concise description of what the issue is. **BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Configuration**
* UTM Version: 
* OS Version: 
* Intel or Apple Silicon? 

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open Console.app, go to `Crash Reports`, and find the latest entry for either QEMU, QEMUHelper, or qemu-\*. Right click and choose `Reveal in Finder`. Attach the report here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by osy** [comment, marked as answer]
No

**Comment by mjonsson1986** [comment]
So What are the deb file for? Sent from Mail for Windows From: osySent: den 25 februari 2022 05:44To: utmapp/UTMCc: mjonsson1986; AuthorSubject: Re: [utmapp/UTM] utm for linux (debian) (Discussion #3678) No—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by leonpano2006** [reply]
I think deb file is for iOS tweaks (for jailbroken iOS)
jailbroken iOS devices uses apt as package manger (please do not add ubuntu or Debian repos as they are linux and iPhone are Darwin. not arch issue)
that deb can't be installed on any linux(distros) that deb is darwin

**Comment by leonpano2006** [comment]
please make version for linux
utm is really good qemu vm manger
is it very hard?
or I need to stuck with outdated aqemu

**Comment by mjonsson1986** [comment]
Eh.. deb and apt are for debian system actualy Skickades från E-post för Windows Från: leonpano2006Skickat: den 15 maj 2022 01:36Till: utmapp/UTMKopia: mjonsson1986; AuthorÄmne: Re: [utmapp/UTM] utm for linux (debian) (Discussion #3678) I think deb file is for iOS tweaks (for jailbroken iOS)jailbroken iOS devices uses apt as package manger (please do not add ubuntu or Debian repos as they are linux and iPhone are Darwin. not arch issue)that deb can't be installed on any linux(distros) that deb is darwin—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by leonpano2006** [reply]
your email blow up format you want to write
login to github to reply is better
